src/utilities/yaml_validation_error.ts

```diff
@@ -0,0 +1,13 @@
+import { IOutputError } from 'better-ajv-errors';
+
+export class YAMLValidationError extends TypeError {
+    constructor(message: string, ajvErrors: IOutputError[]) {
+        super(message);
+        this.name = 'YAML Validation Error';
+
+        console.log(ajvErrors);
```

Comment:
It seems like we should either append  the string versions of the avjErrors to the message, or store them in a member variable, for inspection by whoever catches the YAMLValidationError.